### PR TITLE
Back-port #45582 to 2017.7.3

### PR DESCRIPTION
**Pull request by rallytime**
Back-port #45582 to 2017.7.3

### CODE REVIEW

**Comment by gtmanfred** [comment]
re-run py

File: salt/utils/thin.py [hunk 1]
```diff
@@ -508,6 +508,7 @@ def gen_min(cachedir, extra_mods='', overwrite=False, so_mods='',
         'salt/utils/aggregation.py',
         'salt/utils/yamlloader.py',
         'salt/utils/event.py',
+        'salt/utils/state.py',
```
Comment (gtmanfred) [inline salt/utils/thin.py]: This file doesn't exist in 2017.7.3

**Review by gtmanfred — CHANGES_REQUESTED** [review]
everything else looks good

File: salt/utils/thin.py (hunk 1 above)
Comment (rallytime) [reply salt/utils/thin.py]: Good catch. I've removed that line.

**Review by gtmanfred — APPROVED** [review]

**Pull request merged**